ua_policy_proposal.md

```diff
@@ -59,7 +59,7 @@ For each element of the First Party Set policy, we propose an enforcement method
 <tr>
 <td>A group identity that is easily discoverable by a users </td>
 <td>UI treatment (and co-branding in some cases)<sup>2</sup> </td>
-<td>None (solely the browser's and site author's responsibility)</td>
+<td>Conduct user surveys to determine if common identity is understandable to users</td>
```

Comment:
@dmarti - Do you see this being performed for (a) Every set submission/acceptance, (b) On sets flagged by users and civil society, or (b) Periodically, outside of the set validation process, with the aim of improving criteria/guidance provided in the policy document.

Reply:
@krgovind Added a note on this in a follow-up commit. This probably would not need to happen for every set, but for (b) on some randomly selected sets to calibrate the survey, and as part of handling reports of invalid sets.